### ISSUE TITLE
Research and add archive of pointers to major programming/technical cheatsheets

### ISSUE THREAD

**Issue by rswgnu** [pbellon/cheat#1]
I think this would become much more useful and widely used if you researched the best (in your opinion) cheetsheets that are out there for major programming languages and technical topics.  I know you have a few of these in the sheets directory but this is just skimming the surface.

I don't think you need to create a folder of existing cheetsheets but just provide links to them and have your package retrieve them via the links as requested, then cache them in a local folder and reference from there unless the user requests a 'source' load that would return to the original source for updates.

Think bigger and the payoff in utility and usability will be bigger.

**Comment by pbellon** [comment]
I agree with you, I was planning to add more cheatsheets (from this awesome project https://github.com/rstacruz/cheatsheets/) but I first need to be able to have categories on cheatsheets, that way I'll be able to add a configuration like
```elisp
(setq cheat/categories '("js" "emacs"))
``` 
Which will tell this package to only load the js & emacs categories. I want to do that first because I don't want to overload people with cheatsheets they may not be interested to read. Also I need to be able to convert the markdown cheatsheets into org files with the proper metadata. I think pandoc has all I need for that.

**Comment by rswgnu** [comment]
Sounds good.

-- Bob
